feature/gribi/otg_tests/gribigo_compliance_test/gribigo_compliance_test.go

```diff
@@ -225,3 +290,67 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) gosnappi.Config {
 
 	return top
 }
+
+// AddNHGReferencingToUknownNH tests a case where a nexthop group references a nexthop IP that does not Resolve.
```

Comment:
Suggest to add a comment to this line that says `1.0.0.1` must not resolve within the topology (and does not).

Reply:
Ack